src/utils/parser/tests/fixtures/es6.js

```diff
@@ -0,0 +1 @@
+dispatch({ ...action, [PROMISE]: promise });
```

Comment:
haha - they're not, but the parser doesn't care. it's just doing a `...` rest operation